docs/.doctor-rst.yaml

```diff
@@ -1,8 +1,8 @@
 rules:
     american_english: ~
     avoid_repetetive_words: ~
-    blank_line_after_directive: ~
-    blank_line_before_directive: ~
+    # blank_line_after_directive: ~
+    # blank_line_before_directive: ~
```

Comment:
@OskarStark I disabled these as the `after_directive` was giving a false positive:

e.g.

```
.. codeimport:: ../../examples/Extension/AcmeExtension.php
  :language: php
  :sections: all,report_generator_di
```

results in:

```
.. codeimport:: ../../examples/Extension/AcmeExtension.php
  :language: php
  :sections: all,report_generator_di
```

but having a blank line:

```
.. codeimport:: ../../examples/Extension/AcmeExtension.php

  :language: php
  :sections: all,report_generator_di
```

Gives an error when building because the blank line is classed as `content` and this directive does not allow content.